### ISSUE TITLE
scripts sequence

### ISSUE THREAD

**Issue by Owl64901** [UBC-MDS/Group_7_Project#56]
import_data_split.py would be our first script. So which one is our second script, eda.py or preprocessor_script.py? Note: These scripts must be sequential, meaning that the output of the previous script should be the input of the following script.

**Comment by riyaeliza123** [comment]
Yeah, I agree, lets do eda and then preprocessing haha. We can explain our intuition in the documentation.

**Comment by charlesxch** [comment]
I add several model scripts to the scripts folder. They can be processed in any order. The model_comparison.py should be run after all model.py are done. I am still working on model_comparison.py.
